dashboard/config/videos.csv

```diff
@@ -304,3 +304,15 @@ songwriting_with_functions,,,QuIMLel1mmw,https://videos.code.org/levelbuilder/un
 what_is_an_event,,,HcljJx_0Elw,https://videos.code.org/levelbuilder/disneyinfinityplay-lab-events-mp4.mp4,en-US
 teaching_cs_pair_programming,,,sxToW3ixrwo,https://videos.code.org/levelbuilder/06pairprogramming_small-mp4.mp4,en-US
 teaching_problem_solving,,,XN0G6bSbQ2M,https://videos.code.org/levelbuilder/05teachingproblemsolving_small-mp4.mp4,en-US
+CSF_unspotted_bugs,,,s-9_9Eqx_L4,levelbuilder/leccion-1-unspotted-bugs_es-mp4.mp4,es-MX
```

Comment:
the "download" value for these should be a full url; looks like we're missing the `https://videos.code.org/` prefix

Reply:
Good catch! I've fixed this set and I'll fix up the script itself today.